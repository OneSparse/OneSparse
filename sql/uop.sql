-- <table style="width: 100%; table-layout: fixed;" class="dot-table">
--   <tr>
--     <td style="width: 45%;">
--         <!-- Diagram A -->
\set draw_source :uop_a_source
\i sql/draw.sql
--     </td>
--     <td style="font-size: 30px; vertical-align: middle;">op</td>
--     <td style="width: 45%;">
--         <!-- Diagram B -->
\set draw_source :uop_b_source
\i sql/draw.sql
--     </td>
--   </tr>
-- </table>
