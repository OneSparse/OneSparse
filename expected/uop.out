-- <table style="width: 100%; table-layout: fixed;" class="dot-table">
--   <tr>
--     <td style="width: 45%;">
--         <!-- Diagram A -->
\set draw_source :uop_a_source
\i sql/draw.sql
\setenv draw_source :draw_source
\! echo $draw_source | dot -Tsvg -Nnodesep=0.1 -Nranksep=0.1 -Nwidth=0.05 -Nheight=0.05 -Nfontsize=8 -Efontsize=8 | sed '/<!DOCTYPE/,/>/d; /<?xml/,/>/d' | sed '1i<div>' | sed '$a</div>\n' | sed 's/^/-- /'
Error: <stdin>: syntax error in line 1 near ':'
--     </td>
--     <td style="font-size: 30px; vertical-align: middle;">op</td>
--     <td style="width: 45%;">
--         <!-- Diagram B -->
\set draw_source :uop_b_source
\i sql/draw.sql
\setenv draw_source :draw_source
\! echo $draw_source | dot -Tsvg -Nnodesep=0.1 -Nranksep=0.1 -Nwidth=0.05 -Nheight=0.05 -Nfontsize=8 -Efontsize=8 | sed '/<!DOCTYPE/,/>/d; /<?xml/,/>/d' | sed '1i<div>' | sed '$a</div>\n' | sed 's/^/-- /'
Error: <stdin>: syntax error in line 1 near ':'
--     </td>
--   </tr>
-- </table>
